lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
:+1: Sounds good to me, definitely better than `IncludeableOptions`.
Will update the PR with the requested changes and an `Includables` union type.